src/main/resources/db/migration/0300/V348__FundingEntities.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE accelerator.funding_entities
+(
+    id   BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE TABLE accelerator.funding_entity_projects
+(
+    funding_entity_id BIGINT NOT NULL REFERENCES accelerator.funding_entities,
+    project_id        BIGINT NOT NULL REFERENCES projects
+);
+CREATE INDEX ON accelerator.funding_entity_projects(funding_entity_id);
+CREATE INDEX ON accelerator.funding_entity_projects(project_id);
+
+
+CREATE TABLE accelerator.funding_entity_users
+(
+    funding_entity_id BIGINT NOT NULL REFERENCES accelerator.funding_entities,
+    user_id           BIGINT NOT NULL REFERENCES users
+);
+CREATE INDEX ON accelerator.funding_entity_users(funding_entity_id);
+CREATE INDEX ON accelerator.funding_entity_users(user_id);
```

Comment:
Could make the user ID index `UNIQUE` to enforce the "user can only be in one funding entity" constraint. Or even make user ID the primary key of this table.

Reply:
I like the primary key idea, will add that!